frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -83,7 +83,7 @@ export function useKubeObject<K extends KubeObject>({
   kubeObjectClass,
   namespace,
   name,
-  cluster = getCluster() ?? undefined,
+  cluster = getCluster() ?? '',
```

Comment:
I wonder why '' here instead of undefined?

Reply:
I just moved the default value here from line 100, I'm not exactly sure why empty string is used here